prow/integ-suite-kind.sh

```diff
@@ -113,18 +113,18 @@ elif [[ "$IP_FAMILIES" =~ "IPv6" ]] && [[ "$IP_FAMILIES" =~ "IPv4" ]]; then
    KIND_IP_FAMILY="dual"
 fi
 export KIND_IP_FAMILY
-export NODE_IMAGE="gcr.io/istio-testing/kind-node:v1.31.0"
+export NODE_IMAGE=""
```

Comment:
Why is this being changed?

Reply:
Native sidecars are not enabled on KIND 1.31, so the integration tests for spire worked fine and didn't capture the issue with patching containers, instead of initContainers, so I upgraded the node image, but now other tests fail.